tests/parametric/test_dynamic_configuration.py

```diff
@@ -26,21 +26,23 @@
     "DD_INTERNAL_TELEMETRY_V2_ENABLED": "1",
     # Decrease the heartbeat/poll intervals to speed up the tests
     "DD_TELEMETRY_HEARTBEAT_INTERVAL": "0.2",
-    "DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS": "0.2",
+    "DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS": "1",
```

Comment:
The C++ tracer do not support float input. I would prefer to set the interval to 1 second only for C++ but my knowledge of our framework is limited. Any advice, help would be appreciate! 🙂

Reply:
Not really, at least, it would require the code to be way more complicated than it should.

Reply:
Two solutions here : 

1. the bad one 😃 :  make this value dependant of the tracer
2. the good one : discuss with other teams, and amend the RFC about the supported format for the value

Reply:
Here's my suggestion to move forward:
1. Temporarily make this value dependant of the tracer.
2. Amend the RFC to clarify the environment variable usage.
3. Update the test.

What do you think?

Reply:
fine with me!

Reply:
it's really not a big deal for this case. It is just an arbitrary value less than one so the test can complete quicker.